src/Store/ActionTypes.js

```diff
@@ -19,7 +19,7 @@ export const FILTER_PLATFORM_ITEMS = 'FILTER_PLATFORM_ITEMS';
 export const FETCH_PORTFOLIOS = 'FETCH_PORTFOLIOS';
 export const FETCH_PORTFOLIO_ITEMS = 'FETCH_PORTFOLIO_ITEMS';
 export const FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO = 'FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO';
-export const FETCH_PORTFOLIO_ITEM = 'FETCH_PORTFOLIO_ITEM';
+export const FETCH_PORTFOLIO = 'FETCH_PORTFOLIO';
```

Comment:
If you do such changes, it is also good to remove references also in _src/Store/Reducers/PortfolioStore.js_.